kubejobs/__init__.py

```diff
@@ -548,8 +534,7 @@ def execute(self, data):
 
 def rebuild(app_id, starting_time,
```

Comment:
As you changed the parameters of this function, you need to change any calls to this function too, like this one
https://github.com/ufcg-lsd/asperathos-manager/blob/594ef071e3407c2b1c982fc57aceec2a779c4c86/kubejobs/__init__.py#L118-L131